test/unit/spec/narrative_core/kbaseNarrativeJobsPanel-spec.js

```diff
@@ -4,9 +4,9 @@
 /*global beforeEach, afterEach*/
 /*jslint white: true*/
 define([
-    'kbaseNarrativeJobsPanel'
+    'jobCommChannel'
 ], function(Widget) {
-    describe('Test the kbaseNarrativeJobsPanel widget', function() {
+    describe('Test the jobCommChannel widget', function() {
```

Comment:
I've never actually ran the tests on my local environment, I've just looked at travis believe.

Reply:
Huh, I guess I haven't updated those tests as part of this PR. I can put in a few.

Reply:
You can run tests with either `make test` to do the whole suite, or `make test-backend` or `make test-frontend-unit` to do either/or.